cmd/mattermost/commands/group.go

```diff
@@ -0,0 +1,305 @@
+// Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+package commands
+
+import (
+	"fmt"
+
+	"github.com/mattermost/mattermost-server/model"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var GroupCmd = &cobra.Command{
+	Use:   "group",
+	Short: "Management of groups",
+}
+
+var ChannelGroupCmd = &cobra.Command{
+	Use:   "channel",
+	Short: "Management of channel groups",
+}
+
+var ChannelGroupEnableCmd = &cobra.Command{
+	Use:     "enable [team]:[channel]",
+	Short:   "Enables group constrains in the specified channel",
```

Comment:
"Constrain" is a verb whereas "constraint" is a noun.
```suggestion
	Short:   "Enables group constraint on the specified channel",
```